app/helpers/application_helper.rb

```diff
@@ -1,2 +1,15 @@
 module ApplicationHelper
+  
+  def show_coffee_on_index_page?
```

Comment:
this can be shortened to 
if @most_recent_brew.team_id == current_user.team_id
  return true
else
  return false
end